planner/core/logical_plan_builder.go

```diff
@@ -310,8 +310,17 @@ func (p *LogicalJoin) extractOnCondition(conditions []expression.Expression, der
 	return
 }
 
+// The table alias is only used to check if the logicalPlan match some
```

Comment:
The first line of function's header comment should begin with the function name.
`// extractTableAlias ...`

Reply:
Oh, thanks for reminding me!